src/script/calling/CallingRepository.ts

```diff
@@ -724,11 +724,13 @@ export class CallingRepository {
    * Handle incoming calling events from backend.
    */
   onCallEvent = async (event: CallingEvent, source: string): Promise<void> => {
+    console.info('BARDIA CALLING EVENT', {event, source});
```

Comment:
👀

Reply:
done in https://github.com/wireapp/wire-webapp/pull/18250/commits/15f39798ccc6fa7be3cc2f9d02f2da45b60ae4fe